fsspec/fuse.py

```diff
@@ -164,3 +213,122 @@ def run(fs, path, mount_point, foreground=True, threads=False):
             func()
         except KeyboardInterrupt:
             pass
+
+
+def mount(
+    url,
```

Comment:
Is this meant to be an internal function? If not, some docs would be a good idea.

Reply:
I will merge it with the `main` function.